examples/finite_prime_field.rs

```diff
@@ -0,0 +1,254 @@
+use noether::{
+    AssociativeAddition, AssociativeMultiplication, CommutativeAddition, CommutativeMultiplication,
+    Distributive, FiniteField,
+};
+use num_traits::{Euclid, Inv, One, Zero};
+use std::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Rem, Sub, SubAssign};
```

Comment:
Likewise here, assume Neg is additive negation. (The unfortunate thing is across different textbooks, terminology and notation can differ.)

Reply:
Yeah, I just went off the comments on the underlying traits

Reply:
added comment explaining this in the library.